app/models.py

```diff
@@ -1167,4 +1189,5 @@ def filter_null_value_fields(obj):
 
 
 def generate_new_service_id(framework_slug):
+    # FIXME framework_slug parameter ignored??
```

Comment:
Yes we should fix this.  In the past there were different generation rules for different frameworks.